### PR TITLE
Fix shadow variable warnings

### PR DESCRIPTION
**Pull request by akhillanger**
The global variables are shadowed by function variables/arguments of the same name. This generates compiler warnings. Suffix '_global' at the end of global variables to distinguish them from function arguments/variables.

### CODE REVIEW

**Comment by akhillanger** [comment]
test:jenkins/ch4/ofi

**Comment by akhillanger** [comment]
@raffenet the tests look good. The failure is unrelated.

File: src/mpi/romio/adio/common/onesided_aggregation.c [hunk 1]
```diff
@@ -275,14 +275,14 @@ printf("ADIOI_OneSidedWriteAggregation started on rank %d\n",myrank);
...
+        MPI_Type_extent(datatype, &bufTypeExtent_global);
       }
 
 #ifdef onesidedtrace
-      printf("flatBuf->count is %d bufTypeExtent is %d\n", flatBuf->count,bufTypeExtent);
-      for (i=0;i<flatBuf->count;i++)
-        printf("flatBuf->blocklens[%d] is %d flatBuf->indices[%d] is %ld\n",i,flatBuf->blocklens[i],i,flatBuf->indices[i]);
+      printf("flatBuf->count is %d bufTypeExtent is %d\n", flatBuf_global->count,bufTypeExtent_global);
+      for (i=0;i<flatBuf_global->count;i++)
+        printf("flatBuf->blocklens[%d] is %d flatBuf->indices[%d] is %ld\n",i,flatBuf_global->blocklens[i],i,flatBuf_->indices[i]);
```
Comment (raffenet) [inline src/mpi/romio/adio/common/onesided_aggregation.c]: The last instance looks wrong here in this printf. Also the first reference is not changed to `_global` when it probably should be. This is all inside an `#ifdef` so was not compile tested.

**Comment by raffenet** [comment]
This probably needs review from @pkcoff. Paul, can you take a look?

File: src/mpi/romio/adio/common/onesided_aggregation.c (hunk 1 above)
Comment (akhillanger) [reply src/mpi/romio/adio/common/onesided_aggregation.c]: That is correct. Thanks for catching that. These have been fixed.

**Comment by akhillanger** [comment]
test:jenkins/ch4/ofi

**Pull request merged**